tests/unit/cli/test_main.py

```diff
@@ -336,3 +338,78 @@ def test_project_property_works(
         project1, project2 = n.project, n.project
         assert project1 == project2
         assert from_config_mock.call_count == 1
+
```

Comment:
Well perhaps using a tmpdir is required too. Instead, I choose to mock `os.getcwd` in order to use a directory where a valid LXDock file is present.